open_event/static/js/admin/event/scheduling.js

```diff
@@ -0,0 +1,512 @@
+        $trackElement.find(".track-inner").css("height", time.unit.count * time.unit.pixels + "px");
+        $tracksHolder.append($trackElement);
+    });
+
+    _.each(sessionsStore[dayIndex], function (session) {
+        var $sessionElement = $(sessionTemplate);
+        $sessionElement.data("session-id", session.id);
+        $sessionElement.attr("data-original-text", session.title);
+
+        var scheduled = false;
+        if (!_.isNull(session.start_time) && !_.isNull(session.end_time) && session.start_time != session.end_time) {
+            scheduled = true;
+        }
+
+        if (scheduled) {
+            $sessionElement.addClass("scheduled");
+            $sessionElement.attr("data-original-title", session.start_time + " to " + session.end_time);
+            updateColor($sessionElement);
+
+            if (_.isNull(session.track_id)) {
+                session.track_id = 0;
+            }
+
+            $sessionElement.data("top", session.top);
+            $sessionElement.css("top", session.top + "px");
+            $sessionElement.css("height", minutesToPixels(session.duration) + "px");
+            $tracksHolder.find(".track[data-track-id=" + session.track_id + "]").append($sessionElement);
+        } else {
+            $sessionElement.addClass("unscheduled");
+            $unscheduledSessionsHolder.append($sessionElement);
+            $(".no-sessions-info").hide();
+        }
+
+        $sessionElement.ellipsis().ellipsis();
+
+    });
+    fixOverlaps();
+    $("[data-toggle=tooltip]").tooltip();
+}
+
+function loadData(eventId, callback) {
+    $.get("https://open-event.herokuapp.com/api/v2/events/" + eventId + "/tracks", function (tracks) {
+        $.get("https://open-event.herokuapp.com/api/v2/events/" + eventId + "/sessions", function (sessions) {
+            processTrackSession(tracks, sessions, callback);
+        });
+    });
+}
+
+$(document).on("click", ".date-change-btn", function () {
+    $(this).addClass("active");
+    loadTracksToTimeline($(this).text());
+    $(this).siblings().removeClass("active");
+});
+
+$(document).on("click", ".session.scheduled > .remove-btn", function () {
+    var $sessionElement = $(this).parent();
+    $sessionElement.appendTo($(".sessions-holder"));
+    $sessionElement.addClass("unscheduled").removeClass("scheduled").tooltip("hide").attr("data-original-title", "");
+    $sessionElement.css({
+        "-webkit-transform": "",
+        "transform": "",
+        "background-color": ""
+    }).removeData("x").removeData("y");
+});
+
+function generateTimeUnits() {
+    var start = moment.utc().hour(time.start.hours).minute(time.start.minutes).second(0);
+    var end = moment.utc().hour(time.end.hours).minute(time.end.minutes).second(0);
+    var $timeUnitsHolder = $(".timeunits");
+    var timeUnitsCount = 1;
+    while (start <= end) {
+        var timeUnitDiv = $("<div class='timeunit'>" + start.format('HH:mm') + "</div>");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650806054/issues/source?bid=3233864&fileBranchId=3324174#l478)